### PR TITLE
New stability dashboard, for display on the office dashboards

### PR DESCRIPTION
**Pull request by bsmedberg**
r? @chutten @digitarald 

You can see this live at http://bsmedberg.github.io/telemetry-dashboard/crashes/office-dashboard.html

### CODE REVIEW

File: crashes/office-dashboard.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        chart_type: "missing-data",
+        error: err,
+        missing_text: "Failed to load data.",
+      });
+    });
+  get_crashes(326, "d2f89374fbb282f44b299e1267f9c99771f59773").then(
+    function(data) {
+      setup_e10s_graph(data);
+    },
+    function(err) {
```
Comment (chutten) [inline crashes/office-dashboard.js]: Definitely could factor these out to some common function
```js
var showErrorGraph = function(target, err) {
// MG.data_graphic, etc 
};
```
Then use it like
```js
showErrorGraph.bind(this, "#nightly-graph")
```

**Comment by digitarald** [comment]
Related, we are now showing the first version of http://health.graphics/crashes on the platform dashboards, which is atm ADI and UT crash rate for release.

File: crashes/office-dashboard.js (hunk 1 above)
Comment (bsmedberg) [inline crashes/office-dashboard.js]: @chutten Done.

**Comment by chutten** [comment]
LGTM. How's it look to you, @digitarald ?

File: crashes/office-dashboard.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+document.addEventListener("DOMContentLoaded", function() {
+  var s = d3.select("#beta-graph");
+  gWidth = parseInt(s.style("width"));
+  gHeight = parseInt(s.style("height"));
+
+  get_beta_bydate().then(
+    function(data) {
+      setup_channel_graph(data, "beta");
+    },
+    graph_error.bind(undefined, "#beta-graph")
```
Comment (digitarald) [inline crashes/office-dashboard.js]: (*nit*) bind with `null` is easier to comprehend (vs `undefined` as a way to actually check for undefined variables)

File: crashes/office-dashboard.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    height: gHeight,
+    target: "#" + channel + "-graph",
+    x_accessor: 'activity_date',
+    y_accessor: ['app_crash_rate', 'plugin_crash_rate'],
+    legend: ["App crashes - main+content", "Plugin crashes - NPAPI+GMP"],
+    legend_target: "#legend",
+    // baselines: [{value: 10, label: "max app-crash rate"}, {value: 7.5, label: "max plugin-crash rate"}],
+    markers: markers,
+    interpolate: "linear",
+    area: false,
```
Comment (digitarald) [inline crashes/office-dashboard.js]: Refactor a shared options object out (like margins and graphics display) and merge them with `Object.assign` to reduce repeated lines.

**Pull request merged**